modules/global-classes/api.php

```diff
@@ -0,0 +1,204 @@
+<?php
+
+namespace Elementor\Modules\GlobalClasses;
+
+use Elementor\Core\Kits\Documents\Kit;
+use Elementor\Modules\AtomicWidgets\Styles\Style_Schema;
+use Elementor\Modules\AtomicWidgets\Validators\Styles_Validator;
+
+class API {
+	private Repository $repository;
+
+	public function __construct( Kit $kit ) {
+		$this->repository = new Repository( $kit );
+	}
+
+	public function register_hooks() {
+		add_action( 'rest_api_init', fn() => $this->register_routes() );
+	}
+
+	private function all() {
+		try {
+			$all = $this->repository->all();
+		} catch ( \Exception $e ) {
+			return new \WP_Error( 'unexpected_error', 'Getting global classes failed unexpectedly', [ 'status' => 500 ] );
+		}
+
+		return $all->get();
+	}
+
+	private function get( $request ) {
+		$id = $request->get_param( 'id' );
+
+		try {
+			$one = $this->repository->get( $id );
+		} catch ( \Exception $e ) {
+			return new \WP_Error( 'unexpected_error', 'Getting global class failed unexpectedly', [ 'status' => 500 ] );
+		}
+
+		if ( null === $one ) {
+			return new \WP_Error( 'entity_not_found', 'Global class not found', [ 'status' => 404 ] );
+		}
+
+		return $one;
+	}
+
+	private function delete( $request ) {
+		$id = $request->get_param( 'id' );
+
+		try {
+			$one = $this->repository->get( $id );
+		} catch ( \Exception $e ) {
+			return new \WP_Error( 'unexpected_error', 'Deleting global class failed unexpectedly', [ 'status' => 500 ] );
+		}
+
+		if ( null === $one ) {
+			return new \WP_Error( 'entity_not_found', 'Global class not found', [ 'status' => 404 ] );
+		}
+
+		try {
+			$this->repository->delete( $id );
+		} catch ( \Exception $e ) {
+			return new \WP_Error( 'unexpected_error', 'Deleting global class failed unexpectedly', [ 'status' => 500 ] );
+		}
+
+		return new \WP_REST_Response( null, 204 );
+	}
+
+	private function patch( \WP_REST_Request $request ) {
+		$id = $request->get_param( 'id' );
+		$updated = $request->get_params();
```

Comment:
```suggestion
		$values = $request->get_params();
```

Reply:
Maybe you should use `get_body_params` and maybe you should remove the id in `sanitize_callback`